configure.in

```diff
@@ -13,6 +13,7 @@ AC_HEADER_STDC
 PKG_CHECK_MODULES(GTK, [gtk+-2.0 >= 2.0.0])
 PKG_CHECK_MODULES(JACK, [jack >= 0.80.0])
 PKG_CHECK_MODULES(LIBLO, liblo >= 0.24, LO_FOUND="yes", LO_FOUND="no")
+AC_CHECK_LIB([m],[pow])
```

Comment:
Which compiler/platform are you using to build? It's currently warking as-is on nixos
https://github.com/NixOS/nixpkgs/blob/56904d7c423f2b13b37fbd29f39bbb4b52bc7824/pkgs/applications/audio/timemachine/default.nix